src/app/pages/plan-cuentas/inicio-vigencia/form_info_vigencia.ts

```diff
@@ -0,0 +1,52 @@
+export let FORM_INFO_VIGENCIA = {
+    tipo_formulario: 'currency',
+    alertas: true,
+    modelo: 'FormatoCrearVigencia',
+    campos: [
+        {
+            //Falta mirar si el código del centro gestor se puede asignar de forma dinámica
+            etiqueta: 'select',
+            claseGrid: 'col-lg-6 col-md-6 col-sm-6 col-xs-6',
+            nombre: 'CodigoCentroGestor',
+            label_i18n: 'Codigo Centro Gestor',
+            placeholder_i18n: 'Código Centro Gestor',
```

Comment:
Veo que en el label quedo sin tilde pero en el placeholder no lo del código, ya que esto puede suceder de diversas formas en el cliente, sobre todo repetir textos, todos los textos que se muestran al usuario estan definidos en el archivo i18/es.json

ej   {{'FUENTE_FINANCIAMIENTO.saldo_fuente' | translate }}

<img width="692" alt="Screen Shot 2019-12-09 at 10 06 28 PM" src="https://user-images.githubusercontent.com/50924615/70491787-41f02b80-1ad0-11ea-8625-0274c85a024a.png">

Reply:
this.pUpManager.showErrorAlert(this.translate.instant('mod_fuente_error'));